fenster.h

```diff
@@ -231,7 +231,11 @@ FENSTER_API int fenster_open(struct fenster *f) {
   wc.lpszClassName = f->title;
   RegisterClassEx(&wc);
   f->hwnd = CreateWindowEx(WS_EX_CLIENTEDGE, f->title, f->title,
-                           WS_OVERLAPPEDWINDOW, CW_USEDEFAULT, CW_USEDEFAULT,
+<<<<<<< HEAD
+                           (WS_OVERLAPPED | WS_CAPTION | WS_SYSMENU  | WS_MINIMIZEBOX), CW_USEDEFAULT, CW_USEDEFAULT,
+=======
+                           WS_OVERLAPPEDWINDOW ^ WS_THICKFRAME, CW_USEDEFAULT, CW_USEDEFAULT,
+>>>>>>> b437a200a2a7cc51d659c49d59aaa5044b9ec0ee
```

Comment:
@OetkenPurveyorOfCode 
It seems like something went wrong here while fixing a merge conflict...
Pretty sure this line just needs to be:
```c
                           WS_OVERLAPPEDWINDOW ^ WS_THICKFRAME ^ WS_MAXIMIZEBOX, CW_USEDEFAULT, CW_USEDEFAULT,
```

Reply:
Sure, I messed up, because I am not used to working with git.

I am not sure if it would be better to have a way to enable and handle resizing. Initially I wanted to implement resizing, but there were some tradeoffs to be made and it was unclear what the ideal behaviour should be without breaking the existing API.